External/FEXCore/include/FEXCore/IR/IntrusiveIRList.h

```diff
@@ -20,19 +21,12 @@ namespace FEXCore::IR {
  *
  * Can potentially support reallocation if we are smart and make sure to invalidate anything holding a true pointer
  */
-class DualIntrusiveAllocator final {
+class DualIntrusiveAllocator {
```

Comment:
Should this have a virtual destructor? If not, we should use non-public inheritance instead so people can't accidentally bind child objects to `DualIntrusiveAllocator*` pointers.

Reply:
Yes. Resolved.

Reply:
What happened to this? Looks like the destructor is gone again now.